packages/webawesome/src/components/dropdown/dropdown.ts

```diff
@@ -109,6 +109,18 @@ export default class WaDropdown extends WebAwesomeElement {
 
   async updated(changedProperties: PropertyValues) {
     if (changedProperties.has('open')) {
+      const previousOpen = changedProperties.get('open');
+      // check if the previous value is the same
+      // (if they are, do not trigger menu showing / hiding)
+      if (previousOpen === this.open) {
+        return;
+      }
+      // check if we are changing from undefined to false
+      // (if we are, we can skip menu hiding)
+      if (previousOpen === undefined && this.open === false) {
+        return;
+      }
```

Comment:
as mentioned above, when a component is upgraded, we see `open` change from `undefined` to `false`, which would trigger the `hideMenu`, which would then be cancelled, which would set `this.open = true`. 

All of that would result in the menu defaulting open. In order to prevent this, we check here the previous and open state.

Reply:
I appreciate the extra info in these comments. It's super helpful! Thanks again